suite-native/assets/src/assetsSelectors.ts

```diff
@@ -0,0 +1,81 @@
+import { createSelector } from '@reduxjs/toolkit';
+import BigNumber from 'bignumber.js';
+
+import { Network, networksCompatibility, NetworkSymbol } from '@suite-common/wallet-config';
+import { selectAccounts, selectCoins } from '@suite-common/wallet-core';
+import { toFiatCurrency } from '@suite-common/wallet-utils';
+import { selectFiatCurrency } from '@suite-native/module-settings';
+import { FiatCurrencyCode } from '@suite-common/suite-config';
+
+type Assets = Partial<Record<NetworkSymbol, string[]>>;
+
+interface AssetType {
+    symbol: NetworkSymbol;
+    network: Network;
+    assetBalance: BigNumber;
+    fiatBalance: string;
+}
+
+export const selectBalancesPerNetwork = createSelector(selectAccounts, (accounts): Assets => {
+    const assets: Assets = {};
+    accounts.forEach(account => {
+        if (!assets[account.symbol]) {
+            assets[account.symbol] = [];
+        }
+        assets[account.symbol]?.push(account.formattedBalance);
+    });
+
+    return assets;
+});
+
+export const selectNetworksWithAssets = createSelector(
+    selectBalancesPerNetwork,
+    (assets: Assets): NetworkSymbol[] => Object.keys(assets) as NetworkSymbol[],
+);
+
+export const selectAssetsWithBalances = createSelector(
+    [
+        selectNetworksWithAssets,
+        selectBalancesPerNetwork,
+        selectCoins,
+        selectFiatCurrency,
```

Comment:
why do you still have this selector here? you should remove it because you have it as input lower. Also don't forget to remove that depedency from package.json

Reply:
Good catch. There was 5 selectors but only 4 selector params. Typescript didn't complain and I totally missed this. 

Altough the module-settings dependency is still needed because it needs to be used in Assets.tsx for the selector to be able to pass the fiat currency value into this one.